README.md

```diff
@@ -214,7 +214,7 @@ gitlab:
 
 ## Annotations
 
-By default, the plugin automatically shows the project info corresponding to the location of the `catalog.yaml` file. But you could need some time to force another project, you can do it with the annotations `gitlab.com/project-id` or `gitlab.com/project-slug`:
+By default, the plugin automatically shows the project info corresponding to the location of the `catalog.yaml` file. But you could need some time to force another project. The GitLab instance defaults to `gitlab.com` and can use any configured [GitLab integration host](https://backstage.io/docs/integrations/gitlab/locations) with the annotation `gitlab.com/instance`. The project can be set using with the annotations `gitlab.com/project-id` or `gitlab.com/project-slug`:
```

Comment:
I think it is better not to add the annotation `gitlab.com/instance` in the documentation.